README.md

```diff
@@ -31,7 +31,7 @@ As we cannot host one version of this site with github pages (generated from the
 3. `cd` to the clone of this repository
 4. Run `bundle install`
 5. Serve the site locally via `bundle exec jekyll serve`
-6. Visit the site in the browser via http://localhost:4000
+6. Visit the site in the browser via http://localhost:4000/SORSE20
```

Comment:
Wouldn't this just be the base url now, since we are hosting at sorse.github.io?

Reply:
Done with 61ec875, thanks for spotting this.